lib/models/http_exception.dart

```diff
@@ -0,0 +1,10 @@
+class HttpException implements Exception {
```

Comment:
`HttpException` is already implemented in `dart:io`.
Just write `HttpException` and the IDE should automatically import the library for you.

Reply:
changed implementation from locally managed file to using the available one in dart:io